cmake/kokkos_tribits.cmake

```diff
@@ -277,6 +277,18 @@ MACRO(KOKKOS_CONFIGURE_CORE)
       ENDIF()
    ENDFOREACH()
    MESSAGE(STATUS "Kokkos Devices: ${KOKKOS_ENABLED_DEVICES}, Kokkos Backends: ${FWD_BACKEND_LIST}")
+   set(KOKKOS_GIT_REVISION "UNTRACKED")
+
+   execute_process(COMMAND git rev-parse HEAD
+       OUTPUT_VARIABLE KOKKOS_GIT_REVISION
+       WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
+       RESULT_VARIABLE _RET)
+   if(NOT _RET EQUAL 0)
```

Comment:
What can cause non-zero return value?

Reply:
If the copy of Kokkos doesn't come from a git repo

Reply:
Solved by deferring the feature